### PR TITLE
Create mateusz-lapsa-malawski.yml

### PR DESCRIPTION
**Pull request by munhitsu**

### CODE REVIEW

**Comment by JordanHatch** [comment]
@munhitsu could you update the country code from `UK` to `GB`? Thanks 🙂

File: _people/mateusz-lapsa-malawski.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "Mateusz Lapsa-Malawski"
+account: "@munhitsu@fosstodon.org"
+link: "https://fosstodon.org/@munhitsu"
+image: "https://cdn.fosstodon.org/accounts/avatars/109/301/893/503/108/501/original/35d766787dcd01a2.jpg"
+country: UK
```
Comment (munhitsu) [inline _people/mateusz-lapsa-malawski.yml:6]:
```suggestion
country: GB
```

**Comment by munhitsu** [comment]
@JordanHatch done 🙂

**Comment by JordanHatch** [comment]
@munhitsu Thanks!

**Pull request merged**